style_query.css

```diff
@@ -11,47 +11,70 @@ body {
 /* Grid system has been created below */
 .grid-container {
 	display: grid;
-	grid-template-columns: 18vw auto;
-	grid-template-rows: 29vw 7vw 3vw 50vw 0vw 13vw;
-}
-
-/* Style for nav (navigation) */
-.nav1 {
-	background: #000;
-	opacity: ;
-	font-size: 1.35vw;
-	grid-column: 1;
-	grid-row: 1 / span 6;
-	z-index: 1;
+	grid-template-columns: auto;
+	grid-template-rows: 27vw 7vw 3vw 50vw 0vw 13vw;
 }
 
 /* Style for header, which contains the grid system for the navigation bar */
 .header {
 	background-color: lightblue;
-	grid-column: 2;
+	grid-column: 1 / span 2;
 	grid-row: 1;
 	display: grid;
+	grid-template-columns: auto;
+	grid-template-rows: 4.25vw;
+}
+
+/* Style for nav (navigation) */
+.nav {
+	box-sizing: border-box;
+	background: #000;
+	opacity: ;
+	grid-column: 1 / span 2;
+	grid-row: 1;
+	z-index: 1;
 }
 
 /* Style for search_content - search system */
 .search_content{
 	background: #ffb282;
 	opacity: ;
 	font-size: 1.1vw;
-	margin: 0vw 5vw 0vw 5vw;
-	grid-column: 2;
+	margin: 0vw 0vw 0vw 0vw;
+	grid-column: 1 / span 2;
 	grid-row: 2;
 	display: grid;
 	z-index: 1;
 }
 
+/* Style for text input */
+input[type=text] {

```

Comment:
Below is where I added style for the text input field,  password input field and submit input button.